Sources/StreamChatUI/ChatMessageList/ChatMessage/ChatMessageContentView.swift

```diff
@@ -282,3 +282,22 @@ open class _ChatMessageContentView<ExtraData: ExtraDataTypes>: View, UIConfigPro
         onThreadTap(message)
     }
 }
+
+public typealias ChatMessageAttachmentContentView = _ChatMessageAttachmentContentView<NoExtraData>
+
+open class _ChatMessageAttachmentContentView<ExtraData: ExtraDataTypes>: _ChatMessageContentView<ExtraData> {
+    private var _messageAttachmentBubbleView: _ChatMessageAttachmentBubbleView<ExtraData>?
+    
+    override public var messageBubbleView: _ChatMessageBubbleView<ExtraData> {
+        if let messageBubbleView = _messageAttachmentBubbleView {
+            return messageBubbleView
+        } else {
+            _messageAttachmentBubbleView = uiConfig
+                .messageList
+                .messageContentSubviews
+                .attachmentBubbleView.init()
+                .withoutAutoresizingMaskConstraints
+            return _messageAttachmentBubbleView!
+        }
+    }
```

Comment:
What's happening here?

Reply:
`messageBubbleView` is a lazy property in superclass, so subclass cannot override it with a stored (lazy) property. I've overcame that limitation by basically emulating the `lazy` property with a (non-overridden) stored property